include/fi_signal.h

```diff
@@ -151,7 +151,7 @@ static inline int fi_epoll_wait(int ep, void **contexts, int max_contexts,
 
 	ret = epoll_wait(ep, events, max_contexts, timeout);
 	if (ret == -1)
-		return -errno;
+		return -ofi_sockerr();
 
 	for (i = 0; i < ret; i++)
 		contexts[i] = events[i].data.ptr;
```

Comment:
I wouldn't use sockerr with epoll calls.  Maybe that's something that can be pushed down in the implementation somewhere?

Reply:
For further purposes when EPOLL will be implemented via IOCPs Windows API, I'm going to use `ofi_syserr()` instead `ofi_sockerr()`